CSVSource/src/main/java/io/vantiq/extsrc/CSVSource/CSV.java

```diff
@@ -377,6 +391,9 @@ void handleExistingFiles(String fileFolderPath) {
             Map<String, Object> body = (Map<String, Object>) request.get(BODY_KEYWORD);
             checkedAttribute = PATH_KEYWORD;
             pathStr = (String) body.get(PATH_KEYWORD);
+            if (isRunningInDocker) {
```

Comment:
again, you don't really care about running in docker -- you care about things being unix or windows style file paths.  Please make _that_ clear.

Reply:
done, change to isRunningInLinux